yt/utilities/parameter_file_storage.py

```diff
@@ -8,7 +8,16 @@
     parallel_simple_proxy,
 )
 
-_field_names = ("hash", "bn", "fp", "tt", "ctid", "class_name", "last_seen")
+_field_names = (
+    "hash",
+    "bn",
+    "fp",
+    "tt",
+    "ctid",
+    "class_name",
+    "module_name",
```

Comment:
this is the potentially backwards incompatible part: when using the on-disk store, this new field will be a new column in the csv. so if an on-disk store already exists, there will be an error due to the different number of columns and the user would need to remove the old on-disk store.